scripts/build.ts

```diff
@@ -73,7 +73,6 @@ function _tar(out: string, dir: string) {
       strict: true,
       portable: true,
       cwd: dir,
-      prefix: 'package/',
```

Comment:
I don't understand this one... this puts everything within a `package` directory in the tar, but the real npm packages do not have that directory. Why is this here and why does it work when the real npm packages don't have it?

It was added [to fix a windows issue](https://github.com/angular/angular-cli/commit/f5fe1b60d42b3aa7359b592b5e917a3b41b11398)? But the public packages (such as [cli](https://unpkg.com/browse/@angular/cli@14.0.0/) don't have the directory?